src/web/lib/decidePalette.ts

```diff
@@ -144,9 +144,18 @@ const textArticleLinkHover = (format: Format): string => {
 
 const backgroundArticle = (format: Format): string => {
 	// Order matters. We want comment special report pieces to have the opinion background
-	if (format.design === Design.Comment) return opinion[800];
-	if (format.theme === Special.SpecialReport) return specialReport[800];
-	return 'transparent';
+	if (
+		format.design !== Design.Comment &&
+		format.theme === Special.SpecialReport
+	)
+		return specialReport[800];
+	switch (format.design) {
+		case Design.Comment:
+		case Design.GuardianView:
+			return opinion[800];
+		default:
+			return 'transparent';
+	}
```

Comment:
is this the same as?
```
	if (format.design === Design.Comment) return opinion[800];
	if (format.theme === Special.SpecialReport) return specialReport[800];
	if (format.design === Design.GuardianView) return opinion[800];
	return 'transparent';
```
Neater, but debatable which is easier to read

Reply:
I was first! 😆